content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -38,28 +38,50 @@ Copy the following file into an empty directory and call it `index.html`.
 
 ### Installation
 
+Let's initialize the the package and config files for the example project.
+
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+$ tsc --init
+{{< /tab >}}
+
+{{< tab >}}
+$ npm init -y
+{{< /tab >}}
+
+{{< /tabs >}}
+
 To create traces in the browser, you will need `@opentelemetry/sdk-trace-web`, and the instrumentation `@opentelemetry/instrumentation-document-load`:
 
 ```shell
-npm init -y
-npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
+$ npm install  @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
```

Comment:
- If @svrnm et all feel that you want to keep the init command then add this back.
- Also, drop the `$` since this is a `shell` block

```suggestion
npm init -y
npm install  @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
```

Reply:
as said, I prefer having that npm init -y there, it's not really necessary, but it helps with a few corner cases, especially when you have node_modules anywhere in a parent directory.

@cartermp , wdyt?

Reply:
I'm ok with the npm init.